### PR TITLE
Fix different issues

### PR DESCRIPTION
**Pull request by praveenkumar**
- #15 openshift_option uses bad choice of variable names
- #14 openshift_option uses a hard coded IP
- #7 OpenShift service log entries are to verbose
- #6 OpenShift service script is not stoppable
- #4 Provisioning script does "too much" for a general purpose OpenShift setup
- #3 The OpenShift service creates an OpenShift instance with no authentication

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@praveenkumar I dont think https://github.com/projectatomic/adb-utils/issues/14is addressed in the PR, can you please confirm?

**Comment by praveenkumar** [comment]
@LalatenduMohanty yes it does so now we get IP in `openshift` script but I am still keeping openshift_option as it is in case below step not able to find out IP.

`ip addr show eth1 | awk 'NR==3 {print $2}' | cut -f1 -d /`

File: services/openshift/scripts/openshift [hunk 1]
```diff
@@ -113,15 +112,22 @@ if [ ! -f $master_config ]; then
     done
 
     echo "[INFO] Preparing OpenShift config"
-    start_openshift --write-config=${ORIGIN_DIR}/openshift.local.config > /dev/null 2>&1
+    start_openshift -d --write-config=${ORIGIN_DIR}/openshift.local.config > /dev/null 2>&1
     wait_for_config_files ${master_config} ${node_config}
 
+    echo "Create openshift-dev and admin User"
```
Comment (LalatenduMohanty) [inline services/openshift/scripts/openshift:118]: I think it should be dev in place of openshift-dev. Not a deal breaker, but typing dev is lot easier than openshift-dev

**Comment by LalatenduMohanty** [comment]
@praveenkumar ok, I am also not sure how we are handling #4

File: services/openshift/scripts/openshift (hunk 1 above)
Comment (praveenkumar) [inline services/openshift/scripts/openshift:118]: That is something @hardy suggested to use. I was going with dev but he told me to use openshift-dev.

**Comment by LalatenduMohanty** [comment]
LGTM

**Comment by hferentschik** [comment]
I think I said this before, but it would help to have one pull request per issue. Makes it easier to review and track changes.

**Pull request merged**